package.json

```diff
@@ -1,55 +1,38 @@
 {
-  "author": "Jared Allard <jaredallard@outlook.com>",
   "name": "nexe",
-  "description": "create single executables out of your [node/io].js applications",
+  "description": "Create a single executable out of your Node.js application",
   "license": "MIT",
-  "version": "1.1.2",
+  "version": "2.0.0-beta.1",
   "contributors": [
-    {
-      "name": "Criag Condon",
-      "email": "craig.j.condon@gmail.com",
-      "url": "http://crcn.io/"
-    }
+    "Craig Condon <craig.j.condon@gmail.com> (http://crcn.io)",
+    "Jared Allard <jaredallard@outlook.com>",
+    "Caleb Boyd <caleb.boyd@hotmail.com>"
   ],
   "scripts": {
-    "test": "echo node: $(node -v) && node_modules/.bin/mocha --reporter mocha-circleci-reporter test/test.js"
+    "lint": "eslint index.js ./src/**/*",
+    "test": "npm run lint"
```

Comment:
Tests should be required as well.  It can be difficult to use a standard test provider, given this project's monkey patching, but we really need a way to validate this code, and test for regressions in further patches.